ecs/context.go

```diff
@@ -26,14 +26,43 @@ import (
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/defaults"
-	"github.com/pkg/errors"
 	"gopkg.in/ini.v1"
 
 	"github.com/docker/compose-cli/context/store"
 	"github.com/docker/compose-cli/errdefs"
 	"github.com/docker/compose-cli/prompt"
 )
 
+func getEnvVars() ContextParams {
+	c := ContextParams{}
+
+	//check profile env vars
+	profile := os.Getenv("AWS_PROFILE")
+	if profile != "" {
+		c.Profile = profile
+	}
+	// check REGION env vars
+	region := os.Getenv("AWS_REGION")
+	c.Region = region
+	if c.Region == "" {
+		region = os.Getenv("AWS_DEFAULT_REGION")
+		if region == "" {
+			region = "us-east-1"
```

Comment:
if profile is defined here, we should get the region from the profile as in `getRegion(ecsCtx.Profile)`(backend.go L95)